queue_job_batch/models/queue_job.py

```diff
@@ -30,3 +30,18 @@ def write(self, vals):
             # to work with the same batch
             batch.with_delay().check_state()
         return super().write(vals)
+
+    def autovacuum(self):
+        """Delete batches if all jobs are unlink after job autovacuum
+
+        Called from a cron.
+        """
+        super().autovacuum()
```

Comment:
A good practice is to return the call of the super:
```suggestion
        ret = super().autovacuum()
        
        ...
        
        return ret
```